docs/Updating-your-cBioPortal-installation.md

```diff
@@ -30,24 +30,20 @@ Compile your code again. After restarting the webserver the page should now stat
 
 First, make sure you have the DB connection properties correctly set in your portal.properties file (see [DB connection settings here](portal.properties-Reference.md#database-settings)).
 
-**Dependencies:** the migration script is a python script that depends on the `MySQL-python` library. If necessary, you can install it with the following commands (example for Ubuntu):
+**Dependencies:** the migration script is a Python script that depends on the `mysqlclient` library. If necessary, you can install it with the following commands (example for Ubuntu):
 ```console
-sudo apt-get install python-dev libmysqlclient-dev
-sudo pip2 install MySQL-python
+sudo apt-get install python3-dev default-libmysqlclient-dev
+sudo python3 -m pip install mysqlclient
 ```
 
-For Mac OS X, try the following:
+For macOS, try the following:
 
 ```
 brew install mysql-connector-c
-sudo pip2 install MySQL-python
-```
-
-For Mac OS X, you must also add the following to your .bash_profile:
-
-```
-export DYLD_LIBRARY_PATH=/usr/local/mysql/lib:$DYLD_LIBRARY_PATH
+sudo python3 -m pip install mysqlclient
```

Comment:
the `DYLD_LIBRARY_PATH=/usr/local/mysql/lib:$DYLD_LIBRARY_PATH` is no longer necessary for pyhton3 ?

Reply:
It wasn't clear to me where this recommendation came from in the first place, and whether it applies to modern versions of macOS and the library. I instead linked to the documentation of the library itself a few lines down, which states that some default configuration *may*(?) be broken on macOS, and recommends an entirely different fix.